contracts/lib/GranularRoles.sol

```diff
@@ -28,10 +28,28 @@ abstract contract GranularRoles is AccessControl {
     mapping(bytes32 => address[]) internal _rolesAddressesIndexed; // Used to get roles enumeration
     mapping(bytes32 => bool) internal _rolesFrozen;
 
+    event OwnershipTransferred(
+        address indexed previousOwner,
+        address indexed newOwner
+    );
+
     function owner() public view returns (address) {
         return _owner;
     }
 
+    function transferOwnership(address newOwner)
+        external
+        onlyRole(ADMIN_ROLE)
+    {
+        require(newOwner != _owner, "Already the owner");
+        require(msg.sender != _nftPort, "NFTPort cannot transfer ownership");
```

Comment:
This is not enough, only the `_owner` address should be able to call this function. Otherwise `_nftPort` can assign `ADMIN_ROLE` to another wallet and just use that wallet to transfer ownership.

Reply:
Currently it's impossible
Admin role will not pass check of _regularRoleValid func, and ownership update is no more accessible via update

Reply:
Not via `update()` but `ADMIN_ROLE` can call `grantRole` from `AccessControl` and create new non-NFTPort admins via that.

Reply:
Hmm.. But nftPort has no api endpoint for this. Ok, will add check here too, to make sure.

Reply:
Updated grantRole/revokeRole w/check for non-nftPort